src/Login/Login.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
+import { Redirect } from 'react-router-dom'
+
+class Login extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      login: '',
+      password: '',
+      auth: false
+    }
+    this.loginFormHandler = this.loginFormHandler.bind(this)
+    this.passwordFormHandler = this.passwordFormHandler.bind(this)
+    this.authFormSubmitHandler = this.authFormSubmitHandler.bind(this)
+  }
+
+  authFormSubmitHandler (event) {
+    event.preventDefault()
+    localStorage.setItem('token', process.env.REACT_APP_TOKEN)
+    this.setState({
+      auth: true
+    })
+  }
+
+  loginFormHandler (event) {
+    this.setState({
+      login: event.target.value
+    })
+  }
+
+  passwordFormHandler (event) {
+    this.setState({
+      password: event.target.value
+    })
+  }
+
+  render () {
+    const { login, password, auth } = this.state
+    if (auth) {
+      return <Redirect to='/departments' />
+    }
```

Comment:
this should be done in opposite direction. 
If there is no auth redirect to login.